Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -47,6 +50,9 @@ public protocol ProjectPamphletMainCellViewModelOutputs {
   /// Emits a string for the conversion label.
   var conversionLabelText: Signal<String, Never> { get }
 
+  /// Emits a bool to determine that creatorButton should be hidden.
+  var creatorButtonIsHidden: Signal<Bool, Never> { get }
```

Comment:
Not sure if we need this? Isn't the button a subview of `creatorStackView` which will be hidden?

Reply:
It's not 
<img width="316" alt="Screen Shot 2020-03-06 at 2 49 22 PM" src="https://user-images.githubusercontent.com/11362005/76117364-b21c2980-5fb9-11ea-8f12-a584d3f732a9.png">